src/utils.ts

```diff
@@ -8,19 +8,16 @@ import * as semver from 'semver';
 
 export class Utils {
     public static readonly USER_AGENT: string = 'setup-jfrog-cli-github-action/' + require('../package.json').version;
-    public static readonly SERVER_TOKEN_PREFIX: RegExp = /^JF_ARTIFACTORY_.*$/;
-    // Since 1.45.0, 'jfrog rt c' command changed to 'jfrog c add'
-    public static readonly NEW_CONFIG_CLI_VERSION: string = '1.45.0';
+    public static readonly SERVER_TOKEN_PREFIX: RegExp = /^JFROG_SERVER.*$/;
+
     public static readonly CLI_VERSION_ARG: string = 'version';
-    public static readonly MIN_CLI_VERSION: string = '1.29.0';
+    public static readonly MIN_CLI_VERSION: string = '2.6.0';
```

Comment:
Let's have 2.12.0 as the minimal CLI version. Since people will be manually migrating to the new "v2-jf" major version, there's probably no reason to support older CLI versions.